rotkehlchen/chain/ethereum/modules/balancer/decoder.py

```diff
@@ -272,3 +392,6 @@ def enricher_rules(self) -> list[Callable]:
 
     def counterparties(self) -> list[str]:
         return [CPT_BALANCER_V1, CPT_BALANCER_V2]
+
+    def post_decoding_rules(self) -> list[tuple[int, Callable]]:
+        return [(1, self._check_refunds_v1), (0, self._check_deposits_withdrawals_v1)]
```

Comment:
Change the order here to not confuse the reader. The higher the number is the later the function is ran. And explain why you are giving this priority. Why first deposits/withdrawalsand then refunds.

```suggestion
        return [(0, self._check_deposits_withdrawals_v1), (1, self._check_refunds_v1), ]
```

Reply:
Actually this is wrong and this is why I sorted like this. The comment in the interface is wrong. https://github.com/rotki/rotki/blob/6a2a706c295e4a307c3b1f91baf431f52f5dc457/rotkehlchen/chain/evm/decoding/decoder.py#L142 `sort` by default sorts in ascending order and by doing reverse 1 has higher priority than 0

```
>>> a = [0,1,2]
>>> a.sort(reverse=True)
>>> a
[2, 1, 0]
```

Reply:
I have changed it so 0 is the highest priority